web/src/utils/dashboard/convertSQLData.ts

```
@@ -111,6 +115,10 @@ export const convertSQLData = async (
     return { options: null };
   }
 
+  if (panelSchema?.error_config?.trellis_layout_error_message) {
+    delete panelSchema.error_config.trellis_layout_error_message;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)